ui/src/components/SpotifyPlayer/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FC, useState, useEffect } from "react";
+import { ISpotifyPlayerResponse } from "data/entities/Spotify/ISpotifyPlayerResponse";
+import ky from "ky";
+import { ISpotifyPlayerItem } from "data/entities/Spotify/ISpotifyPlayerItem";
+import { ISpotifyPlayerArtist } from "data/entities/Spotify/ISpotifyPlayerArtist";
+
+const SpotifyPlayer = (props: any) => {
```

Comment:
```suggestion
const SpotifyPlayer = ({
  loginToken = '',
  onError = () => { }
}) => {
```

As discussed, this is a better way to ensure typing of the props keys.

You also could do the following if you wanted types but no defaults. It's a little uglier:

```
const SpotifyPlayer = ({
  loginToken,
  onError
}: {
  loginToken: string,
  onError: Function
}) => {
```